Scheduler:  add example for onOptionChanged and edit currentDate, currentView, views.name, views.type (#1517)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxScheduler/1 Configuration/currentView.md

Specifies the currently displayed view. Accepts the view's [name](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/views/name.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#name') or [type](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/views/type.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#type').

---

> The **Scheduler** displays the first view from the [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) array whose **name** or **type** matches the **currentView** value.

Reviewer: First, describe the situation in which this information matters.

```suggestion
When more than one view matches the **currentView** value, the **Scheduler** displays the first matching view from the [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) array.
```

> To subscribe to changes of the current view, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) option.

Reviewer 2: ```suggestion
To subscribe to changes of the current view, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) handler.
```
